test/ZendeskApi_v2.Test/UserTests.cs

```diff
@@ -497,5 +497,69 @@ public void CanBulkDeleteUsers()
             Assert.AreEqual(JobStatusCompleted, jobResponse.JobStatus.Status.ToLower());
             Assert.AreEqual(users.Count, jobResponse.JobStatus.Total);
         }
+
+        [Test]
```

Comment:
How do you plan on handling the rate limit in the test?

https://developer.zendesk.com/rest_api/docs/core/incremental_export#rate-limits
https://developer.zendesk.com/rest_api/docs/core/incremental_export#incremental-sample-export

May be we add a category to the test so if needed we can mark them as do not run? https://github.com/nunit/docs/wiki/Category-Attribute

Reply:
Also, I am a fan fo the Assert.That() method of unit it test assertions.

Reply:
> Rate Limit
Do you run all tests that often? In my daily work, we have used the incremental Ticket Export and we are yet to hit the limit in actual use. 
So gut feeling is wait and see if it becomes a problem and if so it could be built in that it have a "LastExecuted.txt" file in %temp% and if run too often.
But given that the test CanGetIncrementalTicketExportWithGroupsSideLoadPaged have not given an issue I guess this will not either (have same rate limit)

Reply:
>>Regarding Assert.That

I can change my unit-tests to that (I normally use MS-Test and not nUnit and they do not have the Assert.That so not used to that)...

Practical how do I make this change to the PR? (Is it a new PR?) [Sorry: still very noob on this Git and PR thing]

Reply:
Is this style OK (Assert.That feels clunky for null/not Null checks)?

        [Test]
        public void CanGetIncrementalUserExport()
        {
            var incrementalUserExport = api.Users.GetIncrementalUserExport(DateTimeOffset.MinValue);
            Assert.That(incrementalUserExport.Users.Count, Is.GreaterThan(0));
            Assert.IsNull(incrementalUserExport.Organizations);
            Assert.IsNull(incrementalUserExport.Identities);
            Assert.IsNull(incrementalUserExport.Groups);

            var incrementalUserExportNextPage = api.Users.GetIncrementalUserExportNextPage(incrementalUserExport.NextPage);
            Assert.That(incrementalUserExportNextPage.Users.Count, Is.GreaterThan(0));
            Assert.IsNull(incrementalUserExportNextPage.Organizations);
            Assert.IsNull(incrementalUserExportNextPage.Identities);
            Assert.IsNull(incrementalUserExportNextPage.Groups);
        }

Reply:
@rwjdk  for updating the PR just update the branch on your fork and it will auto update the  PR

As for the test cases I like the 
```csharp
        [Test]
        public void CanGetIncrementalUserExport()
        {
            var incrementalUserExport = api.Users.GetIncrementalUserExport(DateTimeOffset.MinValue);
            Assert.That(incrementalUserExport.Users.Count, Is.GreaterThan(0));
            Assert.That(incrementalUserExport.Organizations, Is.Null);
            Assert.That(incrementalUserExport.Identities, Is.Null);
            Assert.That(incrementalUserExport.Groups,  Is.Null);

            var incrementalUserExportNextPage = api.Users.GetIncrementalUserExportNextPage(incrementalUserExport.NextPage);
            Assert.That(incrementalUserExportNextPage.Users.Count, Is.GreaterThan(0));
            Assert.That(incrementalUserExportNextPage.Organizations, Is.Null);
            Assert.That(incrementalUserExportNextPage.Identities, Is.Null);
            Assert.That(incrementalUserExportNextPage.Groups, Is.Null);
        }
```